ui/library/dist/pdf-components.css

```diff
@@ -1,11 +1,3 @@
-/*
-  Notice that any styling changes would not be reflected to the pdf-components.less in scholar:
-  https://github.com/allenai/scholar/blob/main/online/webapp/app/less/vendor/@allenai/pdf-components.less
-  Any changes to this file should be manually synced up to the pdf-components.less currently.
-
-  There is an ongoing work (https://github.com/allenai/pdf-component-library/issues/72)
-  to update the web config in S2 so that the styling changes in this file can be applied automatically.
-*/
```

Comment:
@yensung i did that with the add props for Download Button & Caroline P is fine with it since at the end of the day the next PR is incrementing the package and whatever we currently have in those .css & .js of dist will be used for publishing so its not really matter of the order of execution. 
Ref: https://github.com/allenai/pdf-component-library/pull/113/files

Reply:
I would still suggest not doing that. These two files are compiled files which are basically not readable and could easily get confused when reviewing PR. For people who just wanna try out pdf-component by pulling down compiled css and js to their project, they might get confused by seeing different outcomes if you pushed the latest compiled files but not incrementing the version number.